addon/background.js

```diff
@@ -0,0 +1,206 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global browser */
+
+function log(...args) {
+  // console.log(...["[TESTPILOT v2] (background)"].concat(args)); // eslint-disable-lint no-console
+}
+
+const storage = browser.storage.local;
+const RESOURCE_UPDATE_INTERVAL = 10000; // 4 hours

```

Comment:
`10000` isn't actually 4 hours ;)

Reply:
oh that was a testing mistake. forgot to change, thanks for catching that!